fastlane_core/lib/fastlane_core/configuration/config_item.rb

```diff
@@ -32,7 +32,7 @@ class ConfigItem
 
     # An optional block which is called when a new value is set.
     #   Check value is valid. This could be type checks or if a folder/file exists
-    #   You have to raise a specific exception if something goes wrong. Append .red after the string
+    #   You have to raise a specific exception if something goes wrong. Use `user_error!` for the message
```

Comment:
Good find 👀 Do we want to give a code example for anyone who might not know what `user_error!` is? Something like `UI.user_error!("your message")`?